addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -1231,6 +1231,7 @@ body.editor_enable.editor_has_snippets {
             overflow-y: auto;
 
             table {
+                @include o-input-number-no-arrows();
```

Comment:
Strange, we already use number input for the donation snippet but I don't see the arrows... how did it work without this rule? In any case, maybe this rule should be global to the whole panel, even out of we-matrix.